metadatastore/commands.py

```diff
@@ -155,8 +157,8 @@ def insert_run_start(time, beamline_id, beamline_config=None, owner=None,
 
 
 @_ensure_connection
-def insert_run_stop(run_start, time, exit_status='success',
-                    reason=None, uid=None):
+def insert_run_stop(run_start, time, uid=None, exit_status='success',
```

Comment:
I don't agree that optional args should always be None. It's a good pattern, but it's not the only good pattern. One advantage of *not* doing it that way is that the user can see from the signature what the default is. I think that makes sense here.